src/katt_callbacks.erl

```diff
@@ -241,7 +245,30 @@ http_request( #katt_request{ method = Method
   Hdrs1 = proplists:delete("x-katt-sleep", Hdrs0),
   Hdrs = proplists:delete("x-katt-timeout", Hdrs1),
   timer:sleep(Sleep),
-  lhttpc:request(Url, Method, Hdrs, Body, Timeout, []).
+  %% need to module-prefixed call for meck's case
+  %% http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+  katt_callbacks:http_request_now(Url, Method, Hdrs, Body, Timeout, []).
+
+http_request_now(Url, Method, Hdrs, Body, Timeout, []) ->
```

Comment:
Re: util - I don't see a reason why a wrapper around an http client shouldn't be in some *_util, except for when it would go to its own module. Apart from that, why "external"?

Re: fusco - we switched to it for almost any of our internal and opensource projects, it's probably not the worst. I think it can be regarded as the official successor of lhttpc. That said, nothing against using hackney either.

I could offer to clean that up if you could be more clear what has to be done besides replacing lhttpc. What's wrong with the tests?